### PR TITLE
[Web API] Base for 6.5 docs

### PR DESCRIPTION
**Pull request by pkosko**
[Web API] Prepared docs structure for 6.5 APIs
* copied all files from 6.0 directory to 6.5
* modified symlink latest to point to 6.5
* modified index.html files to point to 6.5 directory

[Web API] Updated content of Web API reference
Changes contains:
* changing html's metadatas
* hiding all APIs deprecated 2 versions earlier
  * bluetooth
  * content
  * humanactivitymonitor
  * mediacontroller

### CODE REVIEW

**Comment by annie-samsung** [comment]
@mijin85cho, how do we proceed with this PR?

**Review by mijin85cho — COMMENTED** [review]
@annie-samsung To set up the Tizen 6.5 Web API files so that the PRs for Tizen 6.5 Web API can be raised,
copying the Tizen 6.0 Web API files to Tizen 6.5 Web API folder and updating the values to 6.5 would be enough.

Beside, there is one point that can be updated, so I left comment.
If the comment is resolved, would you please approve and merge this PR?

File: docs/application/web/api/latest [hunk 1]
```diff
@@ -1 +1 @@
-6.0
\ No newline at end of file
+6.5
```
[the remark this answers is not in the document]
Comment (pkosko) [reply docs/application/web/api/latest:1]: I've pushed additional commit
https://github.com/Samsung/tizen-docs/pull/1406/commits/cc19988f5fafe5bd76485f4e6d9a713861f220fc 
to this PR

[the remark this answers is not in the document]
Comment (annie-samsung) [reply docs/application/web/api/latest:1]: This concern is addressed hence approving.

**Pull request merged**